.github/workflows/ci.yml

```diff
@@ -113,39 +125,80 @@ jobs:
         run: "cp development/creds.example.env development/creds.env"
       - name: "Linting: pylint"
         run: "poetry run invoke pylint"
+  check-migrations:
+    needs:
+      - "bandit"
+      - "pydocstyle"
+      - "flake8"
+      - "poetry"
+      - "yamllint"
+      - "black"
+    runs-on: "ubuntu-22.04"
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.11"]
+        nautobot-version: ["2.0.0"]
+    env:
+      INVOKE_NAUTOBOT_DATA_VALIDATION_ENGINE_PYTHON_VER: "${{ matrix.python-version }}"
+      INVOKE_NAUTOBOT_DATA_VALIDATION_ENGINE_NAUTOBOT_VER: "${{ matrix.nautobot-version }}"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v4"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v4"
+      - name: "Set up Docker Buildx"
+        id: "buildx"
+        uses: "docker/setup-buildx-action@v3"
+      - name: "Build"
+        uses: "docker/build-push-action@v5"
+        with:
+          builder: "${{ steps.buildx.outputs.name }}"
+          context: "./"
+          push: false
+          load: true
+          tags: "${{ env.PLUGIN_NAME }}/nautobot:${{ matrix.nautobot-version }}-py${{ matrix.python-version }}"
+          file: "./development/Dockerfile"
+          cache-from: "type=gha,scope=${{ matrix.nautobot-version }}-py${{ matrix.python-version }}"
+          cache-to: "type=gha,scope=${{ matrix.nautobot-version }}-py${{ matrix.python-version }}"
+          build-args: |
+            NAUTOBOT_VER=${{ matrix.nautobot-version }}
+            PYTHON_VER=${{ matrix.python-version }}
+      - name: "Copy credentials"
+        run: "cp development/creds.example.env development/creds.env"
+      - name: "Checking: migrations"
+        run: "poetry run invoke check-migrations"
   unittest:
     needs:
       - "pylint"
+      - "check-migrations"
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.8"]
-        db-backend: ["postgresql", "mysql"]
```

Comment:
So, to confirm, we don't want the unit tests to run on:
- postgresql 2.0.0(min nautobot) py3.8
- mysql 2.0.0(min nautobot) py3.8
- mysql 2.0.0(min nautobot) py3.11
- mysql stable py3.8
correct?

Reply:
Yes, correct from my perspective.

@whitej6 Could you please confirm that?

Reply:
Correct, we do not need to test every permutation.